java/src/main/java/com/logicalclocks/hsfs/TrainingDataset.java

```diff
@@ -43,70 +50,24 @@
 import java.io.IOException;
 import java.sql.SQLException;
 import java.text.ParseException;
-import java.util.Date;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
 @NoArgsConstructor
-public class TrainingDataset {
-  @Getter
-  @Setter
-  private Integer id;
-
-  @Getter
-  @Setter
-  private String name;
-
-  @Getter
-  @Setter
-  private Integer version;
-
-  @Getter
-  @Setter
-  private String description;
+public class TrainingDataset extends TrainingDatasetBase {
 
   @Getter
   @Setter
   private DataFormat dataFormat;
 
-  @Getter
-  @Setter
-  private Boolean coalesce;
-
-  @Getter
-  @Setter
-  private TrainingDatasetType trainingDatasetType = TrainingDatasetType.HOPSFS_TRAINING_DATASET;
-
-  @Getter
-  @Setter
-  private List<TrainingDatasetFeature> features;
-
-  @Getter
-  @Setter
-  @JsonIgnore
-  private FeatureStore featureStore;
-
   @Getter
   @Setter
   private StorageConnector storageConnector;
```

Comment:
Why isn't this and other variables like `StatisticsConfig` not saved in the base class?

Reply:
not resolved